spec/services/supplier_chooser_spec.rb

```diff
@@ -0,0 +1,25 @@
+RSpec.describe SupplierChooser do
+  subject(:service) { described_class.new(doorkeeper_application_owner, move) }
+
+  let(:move) { instance_double('Move', from_location: from_location) }
+  let(:from_location) { instance_double('Location', suppliers: suppliers) }
+  let(:suppliers) { [supplier_one, supplier_two] }
+  let(:supplier_one) { instance_double('Supplier') }
+  let(:supplier_two) { instance_double('Supplier') }
+
```

Comment:
I was wondering if we could simplify this, and test with real objects, with a couple of factories, instead of 5 instance doubles.

Reply:
So I very specifically used instance doubles as we use factories/create rows in pretty much every test we write and this service felt like it definitely didn't need any complex functionality that's provided by ActiveRecord 🤔 .

Reply:
Reduced the number of doubles